src/Lifecycle.ts

```diff
@@ -290,10 +285,17 @@ export interface ILocalStorageSession {
  * may not be valid, as it is not tested for consistency here.
  * @returns {Object} Information about the session - see implementation for variables.
  */
-export function getLocalStorageSessionVars(): ILocalStorageSession {
+export async function getLocalStorageSessionVars(): Promise<ILocalStorageSession> {
     const hsUrl = localStorage.getItem(HOMESERVER_URL_KEY);
     const isUrl = localStorage.getItem(ID_SERVER_URL_KEY);
-    const accessToken = localStorage.getItem("mx_access_token");
+    let accessToken = await idbLoad("account", "mx_access_token");
```

Comment:
Need to handle browsers without indexeddb here, or more specifically firefox in private browsing mode which is the only indexedb-less config in a supported browser)